### ISSUE TITLE
Ripple stitch - minimal distance between lines problem

### ISSUE THREAD

**Issue by danielkschneider** [inkstitch/inkstitch#2352]
I got a case where distances between lines seem to be wrong (unless I misunderstood what this param is supposed to do...)

Ink/Stitch Version 3.3 / Debian Linux

File with example:

![carrot-twemoji-inkstitch-wt12-ripple](https://github.com/inkstitch/inkstitch/assets/2872643/41c2f283-f342-4d33-8fe3-5e8961b05328)

Screenshot:

![ripple-stitch](https://github.com/inkstitch/inkstitch/assets/2872643/b6386218-c7a5-4610-8a6a-826c0bc4fb27)

**Comment by bianchifan** [comment]
That's totally correct!
Whitout the minimum value the distance at the turning points will get to small and the embroidery much too dense.
The minimum value outperforms the value for the number of lines.
I have the same problem several times;)

**Comment by kaalleen** [comment]
Yes, it calculates the line distance at the narrowest spot. Doing short stitches could help in density issues if the narrowest spot is located at the end. But it could also be an interesting feature for ripples to offer various end types ...
